Update Ubuntu install instructions to use install script (#5389)
Kong/docs.konghq.com: app/_src/gateway/install/linux/ubuntu.md

Kong is licensed under an
[Apache 2.0 license](https://github.com/Kong/kong/blob/master/LICENSE).

{:.note}
> This page will install {{ site.base_gateway }} in traditional mode, where it acts as both the Control Plane and Data Plane. Running in this mode may have a small performance impact.
> &nbsp;
> 
> &nbsp;
> 
> We recommend using [Konnect](https://konghq.com/products/kong-konnect/register?utm_medium=referral&utm_source=docs&utm_campaign=install-ubuntu) as your Control Plane to allow your Data Plane to run at maximum performance and decrease your deployment complexity

## Prerequisites

* A supported system with root or [root-equivalent](/gateway/{{page.kong_version}}/production/running-kong/kong-user/) access.
* (Enterprise only) A `license.json` file from Kong

## Installation

The quickest way to get started with {{ site.base_gateway }} is using our install script:

> {% navtabs_ee codeblock %}

Author: The nav tabs don't exist when using `navtabs_ee` as they're controlled by this toggle

![CleanShot 2023-04-21 at 12 07 37](https://user-images.githubusercontent.com/59130/233621345-4ca5ff5b-1a40-4095-afb9-911108f305f7.png)

{% navtab Kong Gateway %}
```bash
bash <(curl -sS https://install-script--getkong.netlify.app/install) -v {{page.versions.ee}}
```
{% endnavtab %}
{% navtab Kong Gateway (OSS) %}
```bash
bash <(curl -sS https://install-script--getkong.netlify.app/install) -p kong -v {{page.versions.ce}}
```
{% endnavtab %}
{% endnavtabs_ee %}

This script will detect your operating system and automatically install the correct package. It'll also install a Postgres database and bootstrap {{ site.base_gateway }} for you.

> If you'd prefer to install just the package, please see the [Package Install](#package-install) section.

Reviewer: I think it's just my lack of understanding, but why would you want to just install the package? Or maybe conversely, why would you want to install Gateway, Postgres, and bootstrap Gateway at the same time with the script? But maybe the target audience would already know the answer to this and wouldn't need it spelled out.

Author: This comes down to the target audience for me. Either:

* They know why they need a package only and have the skills required to manage a package install + external DB
* They don't know why they need a package only install, and the install script is what they should use

If we get more questions on this we can expand the _why_ but for now I'd like to keep the page streamlined

### Verify install

Once the script completes, run the following in the same terminal window:

```bash
curl -i http://localhost:8001
```

You should receive a `200` status code.

### Next steps

> Once Kong Gateway is running, you may want to:

Author: ```suggestion
Once {{site.base_gateway}} is running, you may want to:
```

> Once {{ site.base_gateway }} is running, you may want to:

Reviewer: ```suggestion
Once {{ site.base_gateway }} is running, you may want to do the following:
```

> * [Add your Enterprise license (optional)](/gateway/{{ page.kong_version }}/licenses/deploy).

Reviewer: ```suggestion
* Optional: [Add your Enterprise license](/gateway/{{ page.kong_version }}/licenses/deploy).
```

{{ install_from_repo | indent | replace: " </code>", "</code>" }}

> 4. (Optional) Prevent accidental upgrades by marking the package as `hold`:

Reviewer: ```suggestion
4. Optional: Prevent accidental upgrades by marking the package as `hold`:
```
